src/table.rs

```diff
@@ -117,6 +117,46 @@ impl Table {
         }
     }
 
+    /// Sort Key/Value Pairs of the table using the using the comparison function `compare`.
+    ///
+    /// The comparison function receives two key and value pairs to compare (you can sort by keys or
+    /// values or their combination as needed).
+    pub fn sort_values_by<F>(&mut self, compare: F)
+    where
+        F: Fn(&Key, &Item, &Key, &Item) -> std::cmp::Ordering,
```

Comment:
There a reason you do `Fn` unlike `IndexMap::solrt_by`s `FnMut`?

Reply:
Yeah. The reason is that I tried a bunch of things, but I couldn't get anything with `FnMut` to type check properly.

Reply:
OK, I took another run at it and I was able to get it this time. 👍